src/app/screens/auth/components/loginpage/loginpage.component.html

```diff
@@ -0,0 +1,34 @@
+
+
+<body class="bg-gradient-to-r from-pink-500 to-yellow-500 flex items-center justify-center min-h-screen">
```

Comment:
No need to give HTML, head or body tag. Use div directly.

Give the common body css at root level

Reply:
okok i will do the changes now

Reply:
Done sir